conf/bblayers.conf.sample

```diff
@@ -14,7 +14,6 @@ BBLAYERS ?= " \
   ##OEROOT##/meta-uav \
   ##OEROOT##/meta-ros \
   ##OEROOT##/meta-intel-aero \
-  ##OEROOT##/meta-intel-aero-connectivity \
```

Comment:
No need to bump this variable. POKY_BBLAYERS_CONF_VERSION refers to the bblayers schema, not the content so could only change between Yocto releases.